### ISSUE TITLE
Muting the bot when someone on the channel talks

### ISSUE THREAD

**Issue by Matis253** [Splamy/TS3AudioBot#390]
A feature that allows setting of muting the bot when someone on the channel talks about XX percent.
It just needs checking if anyone if somebody is talking at all.

**Comment by Bluscream** [comment]
That's built into teamspeak already

**Comment by Splamy** [comment]
Yeah but the other way around. You probably refer to the channel command which dimms *everyone else* when talking. But in this case the request was to dimm the bot when anyone is talking.

**Comment by Bluscream** [comment]
You can easily do that by using priority speaker for everyone except the bot or by using the ducker plugin

**Comment by Matis253** [comment]
Very funny workaround. Imagine, 20 people on channel, 1 bot, and 19 times setting the Priority Speaker for them. xD

**Comment by ShadowFM** [comment]
What about a setting to reduce the volume of the bot if other people are speaking?

**Comment by Splamy** [comment]
That's exactly the request of this issue...

**Comment by ShadowFM** [comment]
He is talking about a way to mute the bot, so it's a bit different ;)

**Comment by Matis253** [comment]
*Reduce Volume by XX percent.
